src/test/java/domain/LottoGameTest.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+class LottoGameTest {
+
+    @Test
+    @DisplayName("객체 정상 생성 확인")
+    void getLottos() {
+        assertThat(new LottoGame()).isNotEqualTo(null);
```

Comment:
```suggestion
        assertThat(new LottoGame()).isNotNull();
```
null 검증은 바로 표현할 수 있습니다 :)